java/src/jmri/jmrix/dccpp/DCCppReply.java

```diff
@@ -650,824 +665,959 @@ public int maxSize() {
     }
 
     public int getLength() {
-        return(myReply.length());
+        return (myReply.length());
     }
 
-    /* Some notes on DCC++ messages and responses...
+    /*
+     * Some notes on DCC++ messages and responses...
      *
-     * Messages that have responses expected:
-     * t : <T REGISTER SPEED DIRECTION>
-     * f : (none)
-     * a : (none)
-     * T : <H ID THROW>
-     * w : (none)
-     * b : (none)
-     * W : <r CALLBACKNUM|CALLBACKSUB|CV CV_Value>
-     * B : <r CALLBACKNUM CALLBACKSUB|CV|Bit CV_Bit_Value>
-     * R : <r CALLBACKNUM|CALLBACKSUB|CV CV_Value>
-     * 1 : <p1>
-     * 0 : <p0>
-     * c : <a CURRENT>
-     * s : Series of status messages...
-     *     <p[0,1]>  Power state
-     *     <T ...>Throttle responses from all registers
-     *     <iDCC++ ... > Base station version and build date
-     *     <H ID ADDR INDEX THROW> All turnout states.
+     * Messages that have responses expected: t : <T REGISTER SPEED DIRECTION> f
+     * : (none) a : (none) T : <H ID THROW> w : (none) b : (none) W : <r
+     * CALLBACKNUM|CALLBACKSUB|CV CV_Value> B : <r CALLBACKNUM
+     * CALLBACKSUB|CV|Bit CV_Bit_Value> R : <r CALLBACKNUM|CALLBACKSUB|CV
+     * CV_Value> 1 : <p1> 0 : <p0> c : <a CURRENT> s : Series of status
+     * messages... <p[0,1]> Power state <T ...>Throttle responses from all
+     * registers <iDCC++ ... > Base station version and build date <H ID ADDR
+     * INDEX THROW> All turnout states.
      *
-     * Unsolicited Replies
-     *   | <Q snum [0,1]> Sensor reply.
-     * Debug messages:
-     * M : (none)
-     * P : (none)
-     * f : <f MEM>
-     * L : <M ... data ... >
+     * Unsolicited Replies | <Q snum [0,1]> Sensor reply. Debug messages: M :
+     * (none) P : (none) f : <f MEM> L : <M ... data ... >
```

Comment:
Is this change intended? It seems that new line characters have been removed.

Reply:
No, not intended. I will fix. Thanks for catching that.